app/src/main/java/com/dilaefendioglu/dogdiscovery/fragment/DetailFragment.kt

```diff
@@ -0,0 +1,49 @@
+package com.dilaefendioglu.dogdiscovery.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.dilaefendioglu.dogdiscovery.R
+import com.dilaefendioglu.dogdiscovery.data.DogFreeResponse
+import com.dilaefendioglu.dogdiscovery.databinding.FragmentDetailBinding
+import com.squareup.picasso.Picasso
+
+class DetailFragment : Fragment() {
+
+    private lateinit var binding: FragmentDetailBinding
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = FragmentDetailBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        // Bundle'dan veriyi alma
+        val dog = arguments?.getSerializable("breed") as? DogFreeResponse
```

Comment:
SafeArgs kullanabiliriz. for ex;
val dog = DogDetailsFragmentArgs.fromBundle(requireArguments()).breed